src/main.h

```diff
@@ -107,6 +107,8 @@ static const bool DEFAULT_TXINDEX = false;
 static const unsigned int DEFAULT_BANSCORE_THRESHOLD = 100;
 
 static const bool DEFAULT_TESTSAFEMODE = false;
+/** Default for -permitrbf */
+static const bool DEFAULT_PERMIT_REPLACEMENT = true;
```

Comment:
Please don't confuse ...

* **opt-in** in this context stands for: optional enable RBF **per transaction**; so a such transaction can be replaced
* `-permitrbf` is a configuration value that allows accepting such transaction into the nodes mempool